recipe/meta.yaml

```diff
@@ -59,6 +65,7 @@ test:
     - grpc.framework.interfaces.face
   commands:
     - python -m pip check
+    - python distribtest.py
```

Comment:
This seems pretty awesome already: is there a `downstream` that would stress test it further, or is it not worth it, given the number of builds? We fairly frequently get flake as it is.

Reply:
Alas can't seem to get a set of deps to actually be installable, arrow is a bit of a pain for this